src/components/MerchSection/MerchSection.tsx

```diff
@@ -1,20 +1,20 @@
 import React from "react";
 import * as S from "./MerchSection.styles";
-import MerchTeamPhoto from "../../images/merch/MerchTeamPhoto.jpeg";
+import MerchCollection from "./Merch";
+import { merchList } from "./MerchData";
+import useViewport from "components/UseViewport";
 
 const MerchSection = () => {
-  // const { width } = useViewport();
-  // const defaultView = width > 600 ? true : false;
+  const { width } = useViewport();
+  const defaultView = width > 600 ? true : false;
 
-  // const getMerch = () => {
-  //   return merchList;
-  // };
+  const getMerch = () => {
```

Comment:
Don't really need this function (just pass in `merchList` directly to `MerchCollection`